src/scripts/components/Accordion.ts

```diff
@@ -1,155 +1,215 @@
 import { ComponentElement } from '@locomotivemtl/component-manager';
 
-export default class Accordion extends HTMLElement {
-    static readonly DURATION = 300;
-    static readonly CLASS_OPEN = 'is-open';
-    private onClickBind: any;
-    private $root: HTMLDetailsElement;
-    private $summary: HTMLElement;
-    private $content: HTMLElement;
-    private $parent: HTMLElement | null;
-    private animation: Animation | null;
-    private isClosing: boolean;
-    private isExpanding: boolean;
+export class Accordion extends HTMLElement {
+    static CLASS_ACTIVE = 'is-active';
+    static EASING = 'cubic-bezier(0.33, 1, 0.68, 1)';
+    static DURATION = 300;
+
+    private $summary: HTMLElement | null = null;
+    private $content: HTMLElement | null = null;
+    private $details: HTMLDetailsElement | null = null;
+    private animation: Animation | null = null;
+    private isClosing: boolean = false;
+    private isExpanding: boolean = false;
 
     constructor() {
         super();
 
-        // Binding
-        this.onClickBind = this.onClick.bind(this);
-
-        // UI
-        this.$root = this.querySelector('details')!;
-        this.$summary = this.querySelector('summary')!;
-        this.$content = this.querySelector('[data-accordion="content"]')!;
-        this.$parent = this.closest('[data-accordion-parent]') || null;
-
-        // Data
-        this.animation = null;
-        this.isClosing = false;
-        this.isExpanding = false;
+        this.$summary = this.querySelector('summary');
+        this.$details = this.querySelector('details');
+        this.$content = this.querySelector('[data-accordion="content"]');
     }
 
     // =============================================================================
     // Lifecycle
     // =============================================================================
     connectedCallback() {
+        if (!this.validateElements()) return;
```

Comment:
Personally, I think braces make early returns more readable.

```suggestion
        if (!this.validateElements()) {
            return;
        }
```